src/veh_type.cpp

```diff
@@ -719,6 +729,12 @@ std::vector<std::string> vpart_info::engine_excludes() const
     return has_flag( VPFLAG_ENGINE ) ? engine_info->exclusions : std::vector<std::string>();
 }
 
+const std::vector<itype_id> vpart_info::engine_fuel_opts() const
```

Comment:
Please don't return a const value
```suggestion
std::vector<itype_id> vpart_info::engine_fuel_opts() const
```

Reply:
Fixed.